resources/config.yaml

```diff
@@ -0,0 +1,21 @@
+# IPv4 address to listen on
```

Comment:
should this be `resources/config.example.yml` to indicate that it should be changed by the end user?

Reply:
yeah that's probably worthwhile 🙂